lib/elixir/pages/Guards.md

```diff
@@ -47,6 +47,9 @@ For reference, the following is a comprehensive list of all expressions allowed
     * `tl/1`
     * `trunc/1`
     * `tuple_size/1`
+  * the following guard-friendly macros (all documented in the `Integer` module):
```

Comment:
Since those are not automatically included by default, I think our wording needs to mention so, maybe by using the same sentence as below: "if imported from the `Integer` module".